test/__snapshots__/test.spec.js.snap

```diff
@@ -107,6 +107,546 @@ exports[`validate config load config file in ESLint to validate all rules are co
     "@babel/semi": [
       "off",
     ],
+    "@stylistic/array-bracket-newline": [
```

Comment:
Even though `eslint-plugin-prettier` cut a new major for this, there appears to be no breaking changes in our config (tested in `web-schumacher-app`) and so we do not need to cut a corresponding new major (as we do with other major updates for configs/plugins).